tests/ballerina-compiler-api-test/src/test/resources/test-src/symbol_lookup_destructure_var_exclusion_test.bal

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+Person {name: personName, age: personAge, ...rest} = {name: "Ballerina", age: 2, "address" : "Col 3"};
+
+UserDefinedError error(message1, basicErrorNo = detail1) = error UserDefinedError("error message one", basicErrorNo = 1);
+
+function test() {
+    [string, int, float] [s, i, f] = ["Fooo", 4, 6.7];
+
```

Comment:
```suggestion
```

Reply:
This line was left to give the cursor position. i.e., visible symbols at line 23